redisinsight/api/provider-interfaces/models/server.ts

```diff
@@ -0,0 +1,6 @@
+export class Server {
```

Comment:
I do not think this is applicable to Redis Cloud. 
(We may want to display information on subscription, but a subscription is different and distinct from server (or cluster))